libs/ballot-interpreter/src/hmpb-rust/interpret.rs

```diff
@@ -513,16 +506,16 @@ pub fn interpret_ballot_card(
                     BallotPageTimingMarkMetadata::Front(front_metadata),
                     BallotPageTimingMarkMetadata::Back(_),
                 ) => (
-                    side_a,
-                    side_b,
+                    (side_a.0, side_a.1.image, side_a.2, side_a.3),
+                    (side_b.0, side_b.1.image, side_b.2, side_b.3),
                     BallotStyleId::from(format!("card-number-{}", front_metadata.card)),
                 ),
                 (
                     BallotPageTimingMarkMetadata::Back(_),
                     BallotPageTimingMarkMetadata::Front(front_metadata),
                 ) => (
-                    side_b,
-                    side_a,
+                    (side_b.0, side_b.1.image, side_b.2, side_b.3),
+                    (side_a.0, side_a.1.image, side_a.2, side_a.3),
```

Comment:
It looks like `side_a` and `side_b` are only used here. I think this bit would be more readable if we never constructed the `side_a` and `side_b` tuples and just used their components directly.